modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -2250,6 +2250,23 @@ You can also set it up to run as xref:background-jobs-selector[a background job]
 
 NOTE: These commands are not available in xref:maintenance-commands[single-user (maintenance) mode].
 
+=== System
+
+To execute xref:configuration/server/background_jobs_configuration.adoc[background jobs] using xref:configuration/server/background_jobs_configuration.adoc#cron[cron], you can use the `system:cron` command, as in the following example:
```

Comment:
Just tested, but there is no `system:cron` command.
There is a `background:cron` command, which is what I think you wanted to refer...

Reply:
It was added in https://github.com/owncloud/core/commit/6d9aad531fb9005b99d42172e17095618b7463be, and is to be available in oC 11.

Reply:
two points:
- is there a possibility to add a since oc11 tag ? (or something like that)
- propose to add text explaining the differenc between `background:cron` and `system:cron`

Reply:
From the help description, there is no difference between the two commands.

Reply:
I've created an "oC11 - Only" milestone and added #1234 to it.

Reply:
I *guess* the one sets the way how a job is executed (from help, `Use cron to run background jobs`)
While the other one executes a job via cron?

Needs claification, ask core guys and if necessary improve the help text in core to make things more clear.

Reply:
As of the 10.3 release, `system:cron` is now available:

```console
 system
  system:cron                         Execute background jobs as cron
```